index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/css/index.css">
+    <title>ZikaZaki-Folio</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="#" class="logo">My Logo</a>
+            <i id="menu" class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <div class="text-holder">
+                <p class="primary-txt">
+                    I'm Zakariya
+                    Glad to see <br>you!
+                </p>
```

Comment:
- [ ] [REQUIRED] Please make a change! The main title (e.g., "I'm Marianne ...") should be a `h1` tag.